gamedata/cbasenpc.txt

```diff
@@ -15,15 +15,24 @@
 				"linux"		"1058"
 				"windows"	"819"
 			}
+			"g_TouchTrace"
+			{
+				"linux"		"29"
+				"windows"	"32"
+			}
+			"g_EntityListPool"
+			{
+				"windows"	"17"
+			}
 			"CNavMesh::m_isLoaded"
 			{
 				"linux"		"52"
 				"windows"	"52"
 			}
-			"CBaseCombatCharacter::EventKilled"
+			"CBaseEntity::Spawn"
```

Comment:
Nitpick: Have you considered grabbing some gamedata offsets directly from `sdkhooks`? Offsets like `Spawn`, `Touch`, `Weapon_Equip`, `Weapon_Drop`, and `OnTakeDamage_Alive` are also in there.

https://github.com/alliedmodders/sourcemod/blob/28a5d4b342a6499bfe55ef23b714cd660661bcf3/gamedata/sdkhooks.games/engine.ep2v.txt#L80

But I understand putting all offsets under one config out of convenience, hence why I'm considering this optional. If you don't want to act on this then you can resolve this thread.

Reply:
SM's gamedata files are not public API and should ideally not be used

Reply:
Was going to answer something similar to @Mikusch yes this is reason why these are in our gamedata. While I've been guilty of doing otherwise elsewhere in the ext, I decided to stop. I will address the sdkhooks/tools use in the rest of the ext in a latter PR.